src/Jackett.Common/Utils/DateTimeUtil.cs

```diff
@@ -9,9 +9,9 @@ public static class DateTimeUtil
         public const string Rfc1123ZPattern = "ddd, dd MMM yyyy HH':'mm':'ss z";
 
         private static readonly Regex _TimeAgoRegexp = new Regex(@"(?i)\bago", RegexOptions.Compiled);
-        private static readonly Regex _TodayRegexp = new Regex(@"(?i)\btoday([\s,]*|$)", RegexOptions.Compiled);
-        private static readonly Regex _TomorrowRegexp = new Regex(@"(?i)\btomorrow([\s,]*|$)", RegexOptions.Compiled);
-        private static readonly Regex _YesterdayRegexp = new Regex(@"(?i)\byesterday([\s,]*|$)", RegexOptions.Compiled);
+        private static readonly Regex _TodayRegexp = new Regex(@"(?i)\btoday(?:[\s,]+(?:at){0,1}\s*|[\s,]*|$)", RegexOptions.Compiled);
```

Comment:
This look good to me, but I didn't test it. This class has to many cases...
Could you add some unit tests for this class? Not having unit/integration test is worrying me a lot because we are changing a lot of code. You don't have to write test for all cases, just for the new code. Eg `today, today 14:00, today at 15:00, Today at 19PM`...

Reply:
I've been meaning to try and increase test coverage anyway, just been holding off till later because I haven't known what is definitely going to stay the same with all the refactoring we've been doing...

I'll make some tests to cover this class for now, and we'll just adjust them if things move later.